contract/src/main/kotlin/com/ritense/valtimo/contract/client/ApacheRequestFactoryCustomizer.kt

```diff
@@ -33,7 +34,7 @@ class ApacheRequestFactoryCustomizer(
         valtimoHttpRestClientConfigurationProperties.connectionRequestTimeout.let {
             apacheRequestFactory.setConnectionRequestTimeout(Duration.ofSeconds(it))
         }
-        restClientBuilder.requestFactory(apacheRequestFactory)
+        restClientBuilder.requestFactory(BufferingClientHttpRequestFactory(apacheRequestFactory))
```

Comment:
How does impact large requests like file upload/downloads?

Reply:
tested with 2 large files, 250mb and 500mb still works fine